### PR TITLE
Fixing 200+ lint errors, updating some dependencies

### PR DESCRIPTION
**Pull request by mlshort**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -603,7 +604,10 @@
         "antlr4ng": "^3.0.4",
         "await-notify": "1.0.1",
         "d3": "7.9.0",
-        "fs-extra": "11.2.0"
+        "fs-extra": "11.2.0",
+        "node-notifier": "^10.0.1",
+        "swc": "^1.0.11",
+        "@swc/core": "^1.7.14"
```
Comment (mike-lischke) [inline package.json:610]: What are these 3 node packages for? I don't see anything in this patch which uses any of them.

File: package.json [hunk 2]
```diff
@@ -595,6 +595,7 @@
         "build-watch": "npm run build -- --watch"
     },
     "dependencies": {
+        "@stylistic/eslint-plugin-ts": "^2.6.4",
```
Comment (mike-lischke) [inline package.json:598]: Please follow the example set by the other deps (no ^ modifier).

**Review by mike-lischke — CHANGES_REQUESTED** [review]

**Comment by mike-lischke** [comment]
@mlshort Please sign the PR (see the DCO details link for an information how to do that).

File: package.json (hunk 1 above)
Comment (mlshort) [reply package.json:610]: `npm ls -all`

listed them as optional dependencies.

File: package.json (hunk 2 above)
Comment (mlshort) [reply package.json:598]: not all of the other deps are without "^"

this was added by the npm tool itself

`npm install @stylistic/eslint-plugin-ts`

Comment (mlshort) [reply package.json:598]: https://github.com/mike-lischke/vscode-antlr4/blob/master/package.json

there is no consisentency regarding "^", so I just trusted the npm tool in this instance

File: package.json (hunk 1 above)
Comment (mike-lischke) [reply package.json:610]: We don't need them so please remove that.

File: package.json (hunk 2 above)
Comment (mike-lischke) [reply package.json:598]: Yes, npm always adds the "^" modifier, still I don't want them. Sometimes I miss an entry, so it still has that and needs to be fixed.

**Comment by mike-lischke** [comment]
@mlshort Can you please fix the remaining issues soon (including the DCO)? I'd like to do a release this weekend.

**Comment by mike-lischke** [comment]
Note: I update a few node modules myself, just enough to make the build green again. However, you have to rebase your patch now.

**Comment by mlshort** [comment]
> @mlshort Can you please fix the remaining issues soon (including the DCO)? I'd like to do a release this weekend.

No.

I was not that insterested in doing a formal PR to begin with.

I suggest you invoke `npm run lint` and fix yourself.

**Comment by mike-lischke** [comment]
fair enough, thanks

**Pull request closed without merging**